maintenance_request_report/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 - TODAY, Escodoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Maintenance Request Report',
+    'summary': """
+        Maintenance Request Report""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Maintenance',
+    'author': 'Escodoo,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/maintenance',
+    'depends': [
+        'maintenance',
```

Comment:
not needed, as maintenance_request_sequence already depends on maintenance

Reply:
For now I removed the dependency on manitenance_request_sequence